src/http/request.cpp

```diff
@@ -62,6 +76,11 @@ void Request::serialize(net::Buffer &buff) {
         buff << "Content-Length: " << std::to_string(body.length()) << "\r\n";
     }
     buff << "\r\n";
+    for (auto s: mk::split(buff.peek(), "\r\n")) {
+        // TODO: break API, bump minor, pass Var<Logger> to this function
+        // such that this log message is printed with the correct logger
```

Comment:
If we add this functionality, we need to remember to bump version to `0.3.0-alpha`